src/main/java/org/apache/sling/scriptingbundle/plugin/processor/ResourceTypeFolderAnalyser.java

```diff
@@ -66,6 +66,8 @@ public Capabilities getCapabilities(@NotNull Path resourceTypeDirectory) {
                                 fileProcessor.processExtendsFile(resourceType, entry, providedCapabilities, requiredCapabilities);
                             } else if (Constants.REQUIRES_FILE.equals(file.toString())) {
                                 fileProcessor.processRequiresFile(entry, requiredCapabilities);
+                            } else if (org.apache.jackrabbit.vault.util.Constants.DOT_CONTENT_XML.equals(file.toString())) {
+                                fileProcessor.processVaultFile(entry, resourceType, providedCapabilities, requiredCapabilities);
```

Comment:
The filename -> resource type translation is different for content packages as it supports escaping: http://jackrabbit.apache.org/filevault/vaultfs.html#Filename_escaping. I guess we need to introduce a flag to say if we process bundles or content packages.

Reply:
Thanks for this! Implemented in https://github.com/apache/sling-scriptingbundle-maven-plugin/commit/c972385.